azkaban-executor/azkaban-emr-jobtype/src/main/java/uk/gov/dwp/dataworks/azkaban/jobtype/EMRStep.java

```diff
@@ -77,7 +79,7 @@
     private static final int BOOT_POLL_ATTEMPTS_MAX_DEFAULT = 5;
     private final CommonMetrics commonMetrics;
     private volatile AzkabanProcess process;
-    private volatile boolean killed = false;
+    private volatile AtomicBoolean killed = new AtomicBoolean(false);
```

Comment:
technically not needed since each job has its own instance of EMRStep.

Reply:
It's so I can pass it into the cluster id method by reference.

Reply:
I figured that, but when I got to this part of the review, I thought the reason was you may have thought the variable needed to be thread-safe. :-)